challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map &:capitalize
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select { |str| str.eql? "dog" }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject { |str| str.eql? "dog" }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.length
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input[0...2]
 end
 
 def fetch_CD_animals(input)
-  #implement your solution here
+  input.select { |str| ('c'..'d').include? str[0] }
```

Comment:
You've already used the select enumerable with the fetch_the_dog method. Try to find another one to use here.